### PR TITLE
CL: Add new channels

### PR DESCRIPTION
**Pull request by sguinetti**
There are a lot of channels in this PR.

### CODE REVIEW

File: channels/cl.m3u [hunk 1]
```diff
@@ -191,3 +191,117 @@ http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ZonaLatina.cl" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://cdn.mitvstatic.com/channels/cl_zona-latina_m.png" group-title="Music",Zona Latina [Offline]
 http://38.131.11.9:1080/play/a00x
+#EXTINF:-1 tvg-id="Contivision.cl" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Contivisión
+http://unlimited6-cl.dps.live/contivision/contivision.smil/contivision/livestream1/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/cl.m3u:195]:
```suggestion
http://unlimited6-cl.dps.live/contivision/contivision.smil/playlist.m3u8
```

File: channels/cl.m3u [hunk 2]
```diff
@@ -191,3 +191,117 @@ http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/GbALFNA.png" group-title="",Sello de Raza
+https://v2.tustreaming.cl/mastermedia/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",CEAC TV (Santiago)
+https://hd.chileservidores.cl:1936/ceactv/ceactv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Caracola TV (Peñalolén)
+https://wifispeed.trapemn.tv:1936/comunales/caracola-tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Geovisión (Iquique)
+https://5fa5de1a545ae.streamlock.net/Geovision/Geovision/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Antofagasta TV (ATV)
+https://unlimited6-cl.dps.live/atv/atv.smil/atv/livestream2/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/cl.m3u:223]:
```suggestion
https://unlimited6-cl.dps.live/atv/atv.smil/playlist.m3u8
```

File: channels/cl.m3u [hunk 3]
```diff
@@ -191,3 +191,117 @@ http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Exprezión TV (EXTV | Los Álamos)
+https://srv3.zcast.com.br/expreszion/expreszion/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Puncón TV
+https://pantera1-100gb-cl-movistar.dps.live/pucontv/pucontv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",TV Osorno
+https://hd.chileservidores.cl:1936/osorno2/live/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Visión TV Frutillar
+https://vivo.solumedia.com:19360/visiontv/visiontv.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Local",Décima TV (Ancud)
+http://unlimited10-cl.dps.live/decimatv/decimatv.smil/decimatv/livestream1/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/cl.m3u:287]:
```suggestion
http://unlimited10-cl.dps.live/decimatv/decimatv.smil/playlist.m3u8
```

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by sguinetti** [comment]
More of 70 channels were added in this PR.

File: channels/cl.m3u [hunk 4]
```diff
@@ -184,12 +184,166 @@ http://v3.tustreaming.cl/utvsc/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Voz de Poder
+https://v2.tustreaming.cl/vozdepoder/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Unidad Evangelica TV
+https://v2.tustreaming.cl/unidadevangelica/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",TNE
+https://v2.tustreaming.cl/tnetv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Canal ISB (Iglesia San Bernardo)
+https://unlimited1-us.dps.live/isb/isb.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Spectrum Channel LGBTQ+
+https://vdohd.cl:1936/8078/8078/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/cl.m3u:349]: I think the server crashed.

Comment (sguinetti) [reply channels/cl.m3u:349]: Why the server crashed? Some streams take a while, but works.

Comment (LaneSh4d0w) [reply channels/cl.m3u:349]: It doesn't answer anymore on my side. Was scanning the server and poof, it crashed while doing it.

Comment (sguinetti) [reply channels/cl.m3u:349]: I think of Spectrum Channel is now geoblocked.

Comment (LaneSh4d0w) [reply channels/cl.m3u:349]: I don't think so, I think that it's a very unstable server.

Comment (sguinetti) [reply channels/cl.m3u:349]: So, what could you recommend if it will come out as [Timeout] anyway?

Comment (LaneSh4d0w) [reply channels/cl.m3u:349]: I must say that in these kind of cases, it's more a server issue than anything. Not 24/7 would be the appropriate tag, even if it's not a channel shutdown.

Comment (sguinetti) [reply channels/cl.m3u:349]: Very fine. I mark as solved and change the tag.

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**